examples/dynet-benchmark/README.md

```diff
@@ -0,0 +1,17 @@
+# dynet-benchmark (last updated Dec 13, 2017)
+
+- [rnnlm-batch](rnnlm-batch.jl): A recurrent neural network language model with mini-batched training.
+- [bilstm-tagger](bilstm-tagger.jl): A tagger that runs a bi-directional LSTM and selects a tag for each word.
+- [bilstm-tagger-withchar](bilstm-tagger-withchar.jl): Similar to bilstm-tagger, but uses characer-based embeddings for unknown words.
+- [trenn](trenn.jl): A text tagger based on tree-structured LSTMs.
+
+This directory contains examples implemented for [dynet-benchmark](https://github.com/neulab/dynet-benchmark) repo. See [DyNet technical report](https://arxiv.org/abs/1701.03980) for the architectural details of the implemented examples.
+
+## Results on K80
```

Comment:
* The Knet results are from cn2, the dynet/chainer results are from their paper? We need to give hardware specs and note if the results are or are not comparable.
* Are these all gpu results?  What are the cpu results?  Not worth putting in?
* In the ilkarman benchmark I made code comparison easy by having clickable code links for all frameworks.  I can do that by reorganizing the table a bit after I pull this.

Reply:
All the results belong to runs on cn2, but chainer results are old, when our machine was named as ai-test. Though, same CUDA version, same Anaconda distribution. There should not be any major difference.

Yes, those are GPU results. I didn't included CPU results because they were so bad compared to the others.